src/matlab/reactormethods.cpp

```diff
@@ -48,6 +48,13 @@ void reactormethods(int nlhs, mxArray* plhs[],
         case 7:
             iok = reactor_setKineticsMgr(i, int(v));
             break;
+        case 8:
+            // This casting of double to bool is recommended by the MathWorks
```

Comment:
OK I'll take the comment out. We shouldn't end up with any weird doubles since the Matlab function only takes a string `'on'` or `'off'`, and then explicitly uses the `true` and `false` Matlab booleans that should have no problem casting as expected, even if they're passed as `double`s into `ctmethods`.